client/cpp/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.6 FATAL_ERROR)
 SET(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/cli)
 
 SET(LIBRARY_DEPENDENCIES
-    protobuf
+    protobuf::libprotobuf
```

Comment:
Nit: what does this do/how is it different from what was there earlier?

Reply:
The small change made it so that I could compile Anna on macOS using a macports install of protobuf.  Without the ::libprotobuf part, it could not find the protobuf dependencies.  I think it has to do with macports using /opt/local instead of /usr/local.